README.MD

```diff
@@ -141,6 +141,63 @@ The TalendAPI context type requires an _apiUrl_ and an _apiKey_ parameter to con
 
 Any other parameters not listed here can be used as substitution variables in files (when the key is used it is replaced by the value during query execution).
 
+---
+## Actions
+When the application runs, it completes three phases (also called actions). 
+The first is the __initiation__ phase, the next is the test __execution__ and finally comes the __assertion__.
+This order never changes and each phase starts only after the previous has finished.
+This part of the documentation details the role of each action and how the events can be customized during the steps.
+
+### Initiation Action
+The initiation action prepares the environment for the upcoming test execution. 
+During this phase the test runner can automatically create the prerequisites of the test.
+For example, it can load data or clean up after a previous run.
+If the initiation is correctly set up, then every condition will be provided to repeat the test.
+
+If the __testSuite.json__ allows autodetect, then Jarvis parses the configuration for the initiation actions from the folder structure of the test case.
```

Comment:
This seems vague. What else should be said about the testSuite.json here?

Reply:
- all possible fields (I found `src/main/resources/test_bqQuery/testSuite.json` with other fields, but that may not be all)
- this states what happens if we allow autodetect. What happens if we do not? Where does the configuration come from if not from the folders? The file itself? There is nothing about it in the documentation

Reply:
If autodetect is not enabled then it is defined in the file itself.  
Also please document the config for different init actions separately
1. BQLoad
2. SQL script execution
3. SFTP load